index.html

```diff
@@ -116,8 +116,16 @@
 							<figcaption>Nikita Gandhi</figcaption>
 						</figure>
 					</a>
-					</li>	
-					
+					</li>
+					<li>
+					<a href="https://in.linkedin.com/in/tapati-das-742704117" target="_blank">
+					        <figure>
+						        <img class"organiser_img" src="images/tapati.jpg"/>
```

Comment:
You missed an equal to sign after class. Due to that the size parameters from CSS code don't apply to the image and it's displaying as a large size image.